packages/core/rslib.config.ts

```diff
@@ -0,0 +1,34 @@
+import { defineConfig } from "@rslib/core";
+import { pluginModuleFederation } from "@module-federation/rsbuild-plugin";
+
+export default defineConfig({
+  lib: [
+    {
+      bundle: false,
```

Comment:
cannot use bundle, looks like it follows webpack mechanism. we have many shared moduels and Idk how to deal with this here